xivo/wsgi/__init__.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import time
+
+from cheroot import wsgi
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPServer_patch(wsgi.Server):
```

Comment:
python naming conventions is to use CamelCase for class/type names.
```suggestion
class PatchedWSGIServer(wsgi.Server):
```

Reply:
Updated